src/main/java/net/mcreator/ui/help/HelpUtils.java

```diff
@@ -38,18 +38,30 @@
 public class HelpUtils {
 
 	public static Component wrapWithHelpButton(IHelpContext context, Component ca) {
-		return wrapWithHelpButton(context, ca, null, SwingConstants.RIGHT);
+		return wrapWithHelpButton(context, new Object[] { }, ca, null, SwingConstants.RIGHT);
 	}
 
 	public static Component wrapWithHelpButton(IHelpContext context, Component ca, int direction) {
-		return wrapWithHelpButton(context, ca, null, direction);
+		return wrapWithHelpButton(context, new Object[] { }, ca, null, direction);
 	}
 
 	public static Component wrapWithHelpButton(IHelpContext context, Component ca, @Nullable Color ac) {
-		return wrapWithHelpButton(context, ca, ac, SwingConstants.RIGHT);
+		return wrapWithHelpButton(context, new Object[] { }, ca, ac, SwingConstants.RIGHT);
 	}
 
-	public static Component wrapWithHelpButton(IHelpContext context, Component ca, @Nullable Color ac, int direction) {
+	public static Component wrapWithHelpButton(IHelpContext context, Object[] contextArguments, Component ca) {
+		return wrapWithHelpButton(context, contextArguments, ca, null, SwingConstants.RIGHT);
+	}
+
+	public static Component wrapWithHelpButton(IHelpContext context, Object[] contextArguments, Component ca, int direction) {
+		return wrapWithHelpButton(context, contextArguments, ca, null, direction);
+	}
+
+	public static Component wrapWithHelpButton(IHelpContext context, Object[] contextArguments, Component ca, @Nullable Color ac) {
+		return wrapWithHelpButton(context, contextArguments, ca, ac, SwingConstants.RIGHT);
+	}
+
+	public static Component wrapWithHelpButton(IHelpContext context, Object[] contextArguments, Component ca, @Nullable Color ac, int direction) {

```

Comment:
`Object[] contextArguments`

This is messy code, use varargs properly

Reply:
I moved to arrays to make the code work. Is **varargs** still needed?

Reply:
Yes, because you can make it work with varargs

Reply:
You could make eg. method with different name for varargs variant so overloads don't interfere with varargs

Reply:
Even better way may be to use IHelpContext to add "registry name" context and a token to access it in templates, that would be optimal I think

Reply:
Is [this](https://github.com/MCreator/MCreator/pull/1379/commits/12d192aa69db9bfaab0c131782e9d65673ba5297) good? Or should I totally refactor `HelpUtils`?

Reply:
I think it is good enough, will review properly in next days ;)